src/main/resources/templates/nonceStateCheck.html

```diff
@@ -231,7 +227,7 @@ <h5 class="alert alert-warning">In a real tool that is not for demonstration pur
             document.getElementById('pmState').value = [[${ expected_state }]];
             document.getElementById('cookies').value = 'true';
             document.getElementById("post_button").disabled = false;
-            windows.alert('Not using post messages, just cookies because this is not an iframe. Values have just been filled in the form to pass this step');
+            window.alert('Not using post messages, just cookies because this is not an iframe. Values have just been filled in the form to pass this step');
```

Comment:
Should this be `console.warn` or `console.log` instead of `window.alert`?

Reply:
I want the user to be aware of it... because if not, the screen does not make any sense!